### ISSUE TITLE
Presentation from beneath navigation bar?

### ISSUE THREAD

**Issue by nick-iCars** [Daltron/NotificationBanner#16]
Not sure if this would be handy for other folks too but for my purposes I need to sho the notification for a longer than usual time ~20 seconds, but I'd rather cover the nav bar buttons / interactions.

**Comment by Daltron** [comment]
Hi @nick-iCars! 😄 

You aren't the first person to propose this functionality so I am going to go ahead and add it to the next release milestone! 

Thanks!

**Comment by nick-iCars** [comment]
😀

**Comment by nick-iCars** [comment]
LMK if I can contribute on anything. Haven't really looked at the code tbh.

**Comment by Daltron** [comment]
@nick-iCars I absolutely will! It shouldn't be too difficult to implement. 
Thanks! 😄

**Comment by Daltron** [comment]
Included in [v1.3.0](https://github.com/Daltron/NotificationBanner/releases/tag/1.3.0) release! 🎉

**Comment by nick-iCars** [comment]
Nice work, dude! Thanks again!

**Comment by nick-iCars** [comment]
Was just looking to update my project using the new features and I'm running into an issue, I was going to just submit a pull request but I'm not 100% sure what is going on, figured I'd ask. So I ran pod update and pod install it looks like it sees the update and installs it (pod install logs: 'Using NotificationBannerSwift (1.3.0)'), however when I try and use the show(from:) method it does not appear to be included. I went in to inspect what was going on and I also saw an error on line 73 of NotificationBanner.swift:
        titleLabel!.type = .left

NotificationBanner.swift:73:29: Type 'MarqueeLabel.MarqueeType' has no member 'left'

I looked at the repo and files on github and I don't see this on the file on master branch (it uses the correct call of titleLabel!.type = .leftRight, but maybe im not understanding something. Any idea what is going on here? Is there some different repo for the cocoapod im not seeing?
